### PR TITLE
add class for arrows when reached edge

### PR DESCRIPTION
**Pull request by RusinovAnton**
I needed custom styling for arrows when scrolling tabs reached edge so I made this changes.

### CODE REVIEW

File: scrolling-tabs.js [hunk 1]
```diff
@@ -631,43 +629,54 @@
...
+        // taking 2px offset error
+        if (stc.movableContainerLeftPos <= 0 && stc.movableContainerLeftPos >= -2) {
+          // left edge
+          edge = 'left';
+        } else if ((stc.movableContainerLeftPos - newMinPos) >= 0
+          && (stc.movableContainerLeftPos - newMinPos) <= 2) {
+          edge = 'right'
+        }
+
+        stc.handleArrowsClassNames(edge);
```
Comment (RusinovAnton) [inline scrolling-tabs.js]: @mikejacobson yeah I've fckd diff with formatting, but here is main diff here

File: scrolling-tabs.js [hunk 2]
```diff
@@ -99,54 +97,54 @@
...
   // prototype methods
-  (function (p){
+  (function (p) {
     p.handleClickOnLeftScrollArrow = function (e) {
       var evh = this,
-          stc = evh.stc;
+        stc = evh.stc;
 
-      stc.scrollMovement.incrementScrollLeft();
+      stc.scrollMovement.incrementScrollLeft()
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Can you restore the ending semicolon here.

File: scrolling-tabs.js [hunk 3]
```diff
@@ -563,9 +559,11 @@
         return;
       }
 
+      console.log(parseInt(stc.scrollingTabsActiveOffset || 0, 10));
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Can you remove this `console.log`.

File: scrolling-tabs.js [hunk 4]
```diff
@@ -590,34 +588,36 @@
         }
 
         smv.slideMovableContainerToLeftPos();
+      } else {
+        stc.handleArrowsClassNames('left');
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Can you add a condition so this function is only called if `scrollingTabsActiveOffset` is set, so it doesn't affect other users.

File: scrolling-tabs.js [hunk 5]
```diff
@@ -631,47 +631,58 @@
           smv.decrementMovableContainerLeftPos(newMinPos);
           stc.$movableContainer.stop().animate({ left: smv.getMovableContainerCssLeftVal() }, 'fast');
         }
+
+        var edge;
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Basically the same comment as above: can you add a condition so this block of code (from here down to the call to `handleArrowsClassNames` on line 645) only executes if `scrollingTabsActiveOffset` is set, so other users aren't affected.

File: scrolling-tabs.js [hunk 6]
```diff
@@ -684,14 +695,16 @@
     stc.scrollMovement = new ScrollMovement(stc);
     stc.eventHandlers = new EventHandlers(stc);
     stc.elementsHandler = new ElementsHandler(stc);
+
+    stc.scrollingTabsActiveOffset = opts.scrollingTabsActiveOffset || 0
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Can you add an ending semicolon here.

File: scrolling-tabs.js [hunk 7]
```diff
@@ -719,24 +732,45 @@
 
         if (!actionsTaken.didScrollToActiveTab) {
           scrollMovement.scrollToActiveTab({
-            isOnTabsRefresh: options.isWatchingTabs
+            isOnTabsRefresh: options.isWatchingTabs,
```
Comment (mikejacobson) [inline scrolling-tabs.js]: Can you remove the trailing comma here.

**Comment by mikejacobson** [comment]
Other than the things I commented on, this looks good. If you can take care of those things, I can merge this in.

Thanks for contributing!

**Pull request closed without merging**